### PR TITLE
[hotfix] Remove raw class usages in Configuration.

### PR DESCRIPTION
**Pull request by dawidwys**
Trivial change removing some raw `Class` usage around Configuration/ConfigOptions.

### CODE REVIEW

**Comment by aljoscha** [comment]
Looks good to merge! The failure is unrelated.

File: flink-core/src/main/java/org/apache/flink/configuration/Configuration.java [hunk 1]
```diff
@@ -909,12 +909,10 @@ private void loggingFallback(FallbackKey fallbackKey, ConfigOption<?> configOpti
 			List<String> listOfRawProperties = StructuredOptionsSplitter.splitEscaped(o.toString(), ',');
 			return listOfRawProperties.stream()
 				.map(s -> StructuredOptionsSplitter.splitEscaped(s, ':'))
-				.map(pair -> {
+				.peek(pair -> {
```
Comment (twalthr) [inline flink-core/src/main/java/org/apache/flink/configuration/Configuration.java:912]: not sure if we should use `peek` here: https://stackoverflow.com/questions/44370676/java-8-peek-vs-map

Comment (dawidwys) [reply flink-core/src/main/java/org/apache/flink/configuration/Configuration.java:912]: Peek and map behave exactly the same in respect to the use cases described in that thread. I'd say the thread discusses the declarative aspect of the stream API. 

In a code:
```
stream()
.map()/peek()
.count()
```
Neither `map` nor `peek` will be executed (depends on the jvm though) as they cannot change the result of `count()`. The linked thread rather compares `forEach` vs `peek`, in my opinion.

In our use case particularly, the purpose of the `peek` is to add a sanity check right before the collection. In this case in my opinion it is absolutely safe to use `peek` and the code with `peek` is no different than previous version with `map` (minus the return value).

Personally, I'd prefer to change it to `peek` as it removes IDE warning.

Comment (twalthr) [reply flink-core/src/main/java/org/apache/flink/configuration/Configuration.java:912]: Thanks for the explanation. +1

**Pull request merged**